src/services/misc.ts

```diff
@@ -0,0 +1,11 @@
+import request from '../utils/request';
+import API from '../constants/api';
+
+export const getSdkVersion = async<T> (key: string) => {
+    const url = `${
+        API.sdkVersionURI
+            .replace(':key', key)
+    }`;
+
+    return request<T>(url);
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/FeatureProbe/feature-probe-ui/pullRequest?prid=9905084)
```suggestion
  return request<T>(url);
```